app/controllers/shared/api.php

```diff
@@ -74,17 +74,20 @@
         ->from($queueForEvents)
         ->trigger();
 
-    $queueForWebhooks
-        ->from($queueForEvents)
-        ->trigger();
 
-    if ($queueForEvents->getProject()->getId() === 'console') {
-        return;
+    /** Trigger webhooks events only if a project has them enabled */
+    if ($project->getAttribute('webhooks', []) !== []) {
+        $queueForWebhooks
+            ->from($queueForEvents)
+            ->trigger();
     }
 
-    $queueForRealtime
-        ->from($queueForEvents)
-        ->trigger();
+    /** Trigger realtime events only for non console events */
```

Comment:
but the console also uses realtime events, or those are all associated with a specific project?

Reply:
We don't fire realtime events for DOCUMENT_CREATE events fired from the console. This is the existing behaviour. I have not changed it. 

https://github.com/appwrite/appwrite/blob/1.6.x/app/controllers/shared/api.php#L81-L88

https://github.com/appwrite/appwrite/blob/1.6.x/app/controllers/shared/api.php#L529-L536